src/main/java/cn/academy/crafting/api/ImagFusorRecipes.java

```diff
@@ -46,6 +46,14 @@ public void addRecipe(IFRecipe recipe) {
 		recipe.id = recipeList.size() - 1;
 	}
 	
+	public void removeRecipe(Item item) {
+		for (IFRecipe r : recipeList) {
+			if (r.consumeType.getItem() == item) {
```

Comment:
as mentioned in ```IFRecipe```:
> * The match rule only applies for stackable, non-damageable type.
> * Currently the machine will IGNORE item damage and treat all item with different damage equally.

Current implementation doesn't account on stack damage. But it's reasonable to use ```ItemStack``` as paramter but not check the damage for interface consistency, thus we can easily change the implementation later without changing the method signature.